model/version_activation.go

```diff
@@ -148,6 +148,13 @@ func ActivateElapsedBuildsAndTasks(v *Version) (bool, error) {
 	// If any variants/tasks were activated, update the stored version so that we don't
 	// attempt to activate them again
 	if hasActivated {
+		if err := v.UpdateActivation(); err != nil {
+			grip.Error(message.WrapError(err, message.Fields{
+				"operation": "version-activation",
```

Comment:
I would still call this operation "project-activation"; it's what ties all the errors from this overall operation together (not sure why the file is "version_activation" and the logs use "project-activation" but)